### DISCUSSION TITLE
nav + (from redo to undo) don't work properly(macos)

### DISCUSSION THREAD

**Discussion by northWindNA** [manna-harbour/miryoku#82] [Q&A] [answered]
Hi, manna-harbour! Your keymap is awesome, but I can't use these commands from nav layer. Somehow cut is working, but such features as redo or undo, or copy/paste don't work. I think that I'm stupid enough to mess this up. My board is jorne, i've used this "qmk flash -kb crkbd -km manna-harbour_miryoku" to flash keyboard. All features are fine except this nav + redo to undo. Maybe I don't understand how to use them? And I've tried to change keymap in Idea, but such combination as nav+undo etc Idea does not catch. Maybe someone has the same problem.

**Comment by bryanprange** [comment, marked as answer]
https://github.com/manna-harbour/miryoku/tree/master/docs/reference#clipboard

The default uses func keys that must be bound per application, seemingly to keep the default state as platform-agnostic as possible. You probably want:

```
qmk flash -kb crkbd -km manna-harbour_miryoku -e MIRYOKU_CLIPBOARD=WIN
```

**Reply by northWindNA** [reply]
I'll try flash it your way. Thank you.

**Reply by northWindNA** [reply]
"qmk flash -kb crkbd -km manna-harbour_miryoku -e MIRYOKU_CLIPBOARD=MAC" yap that worked for me. Thank you again! There is one reason I had problems - my stupidity and blindness (◐‿◑)﻿
